variables.tf

```diff
@@ -306,6 +306,12 @@ variable "route53_zone_dns_name" {
   }
 }
 
+variable "graphdb_external_dns" {
+  description = "External address where GraphDB will be accessed"
```

Comment:
Update the description to make it more clear as it is still relevant for the old var name.